phsa-adapter/src/BasicAuthenticationHandler.cs

```diff
@@ -1,63 +0,0 @@
-﻿using System;
-using System.Security.Claims;
```

Comment:
The basic auth code is needed for the non-FHIR inbound payload from PHSA

Reply:
restored